src/lib/documents.ts

```diff
@@ -3,25 +3,60 @@ import { isDate } from './utils';
 
 export const duiRegExp = /(^\d{8})-(\d$)/;
 export const nitRegExp = /(^\d{4})-(\d{6})-(\d{3})-(\d$)/;
+export const passportRegExp = /^[A|B|D]\d{8}$/;
 
 /**
- * Verifies that given DUI format is valid
+ * Verifies that given DUI digits are valid for a specific verifier
  *
- * @param  {string} str       A string representing DUI digits (with hyphen)
+ * @param  {number} digits    A number representing DUI digits (8 digits)
+ * @param  {number} verifier  A number representing DUI verifier (1 digit)
  * @returns {boolean}         Validity of the given DUI
  */
-export function isDUI(str: string): boolean {
-  if (!duiRegExp.test(str)) return false;
-
+function isValidDUI(digits: number, verifier: number): boolean {
   let sum = 0;
-  const [digits, verifier] = str.split('-');
-  for (let i = 0; i < digits.length; i++) {
-    sum += Number(digits[i]) * (digits.length + 1 - i);
+  for (let i = 0; i < String(digits).length; i++) {
+    sum += Number(String(digits)[i]) * (String(digits).length + 1 - i);
   }
 
   return Number(verifier) === (10 - (sum % 10)) % 10 && sum > 0;
 }
 
+/**
+ * Verifies that given DUI with hyphen is valid
+ *
+ * @param  {string} str       A string representing DUI digits (with hyphen)
+ * @returns {boolean}         Validity of the given DUI
+ */
+function isDUIWithHyphen(str: string): boolean {
+  if (!duiRegExp.test(str)) return false;
+  const [digits, verifier] = str.split('-');
+  return isValidDUI(Number(digits), Number(verifier));
+}
+
+/**
+ * Verifies that given DUI in numeric format is valid
+ *
+ * @param  {string} str       A string representing DUI digits (without hyphen)
+ * @returns {boolean}         Validity of the given DUI
+ */
+function isDUIWithNumbers(str: string): boolean {
+  if (!(str.length === 9)) return false;
```

Comment:
Can you please:

1. Submit this feature in a separate PR
2. Evaluate the string using RegExp, instead of just validating the length of the string. In this way, we are going to be able to verify that the string only contains digits